packages/@uppy/locales/src/fi_FI.js

```diff
@@ -141,7 +141,6 @@ fi_FI.strings = {
   streamActive: 'Jako aktiivinen',
   streamPassive: 'Jako passiivinen',
   micDisabled: 'Käyttäjä on estänyt mikrofonin',
-  micIsOn: 'Mikrofoni on päällä',
```

Comment:
Intended @arturi?

Reply:
Yes, half of the strings were actually unused in the UI. Am I right @jukakoski?

Reply:
@arturi I have to check. I used webcam feature as basis for this feature and in that component there is an info screen telling if some permissions are missing. At some point there was also that kind of screen in this one which was stripped off later on.

Reply:
It seems "selectSourceTitle: 'Valitse kaappauksen kohde'" is not used as native UI in browser is taking care of selecting the source.
"selectSourceDescription: 'Salli ruudun jako, jotta tallennus on mahdollista.'," this same thing.
Also micIsOn is not in use. Actually at the moment there is no information for the user that screen capturing is capturing mic as well. It should be optional to select audio source or disable audio. 
Todo list stuff?